devbodaghe/facebook-for-woocommerce
# T208173068: Add Prettier Dependency to Codebase

# Add Prettier and Format Existing Files

## Summary
This pull request introduces **Prettier** to the project for code formatting and ensures all existing files adhere to the newly established formatting rules.

### Key Changes:
1. Added **Prettier** to the project dependencies.
2. Created a `.prettierrc` configuration file with project-specific formatting rules.
3. Updated the `package.json` to include formatting scripts:
   - `format`: Formats all files in the project.
   - `format:check`: Checks if files comply with Prettier formatting.
4. Ran Prettier on all existing files to enforce consistent formatting.

## Why This Is Needed
- Establishes a uniform code style across the project.
- Improves code readability and maintainability.
- Simplifies collaboration by reducing style-based merge conflicts.

## How to Test
1. Install the updated dependencies:
   ```bash
   npm install

2. Check formatting:
   ```bash
   npm run format

3. Apply formatting (if needed):
   ```bash
   npm run format:check
